### PR TITLE
Add a gitignore

### PR DESCRIPTION
**Pull request by SCKelemen**
Adding a gitignore will help keep the repo clean. Large projects can become cumbersome and messy, if we don't limit items.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,353 @@
+## Ignore Visual Studio temporary files, build results, and
```
Comment (gondar) [inline .gitignore]: too many comments

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+## files generated by popular Visual Studio add-ons.
+# User-specific files
+*.suo
```
Comment (gondar) [inline .gitignore]: should ignore only files that are specific to used programming approach

**Comment by gservat** [comment]
`.gitignore` should contain one line, and one line only:

`*`

**Comment by iennae** [comment]
duplicate of #367 #48 #432 #91

**Comment by SCKelemen** [comment]
Good bye, cruel world.

**Pull request closed without merging**